Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM node:12.18.0
+FROM node:12.18.0 as builder
 LABEL maintainer="Automattic"
 
-WORKDIR    /calypso
+WORKDIR /calypso
```

Comment:
At some point my editor removed the alignment spaces and I didn't bother adding them back... let me know if you think I should bring them back.

Reply:
I think I was playing with converting space to tabs and automatic indentation of `RUN` commands that spawn multiple lines, but not 100% sure.